assets/blocks/quiz/question-block/question-block.editor.scss

```diff
@@ -108,7 +108,7 @@ $block: '.sensei-lms-question-block';
 	}
 
 	&__answer--multiple-choice {
-		&__option.is-draft {
+		&__option {
```

Comment:
We could hide this button for draft questions reduce some clutter.

Reply:
Done in https://github.com/Automattic/sensei/pull/4031/commits/dc663cf0683f114aadb6fc164ad7a3b8bee863ae